frontend/src/features/frame/components/sidebar/SideBar.tsx

```diff
@@ -121,6 +129,7 @@ const SideBar = () => {
     >
       <List onClick={isSmallScreen ? () => dispatch(closeDrawer()) : undefined}>
         {menuItems.map(({ targetUrl, IconComponent, displayText }) => {
+          if (!IconComponent || !targetUrl) return <Divider />;
```

Comment:
This causes a minor React warning (relevant only for optimization purposes): 
```console
Warning: Each child in a list should have a unique "key" prop.
```

`key={index}` could be used in the Divider.